data/footerAnchor.js

```diff
@@ -0,0 +1,16 @@
+const anchors= [
+    {
+        link: "https://twitter.com/apeunit",
+        content: "Twitter",
+    },
+    {
+        link: "https://www.facebook.com/apeunit/",
+        content: "Facebook",
+    },
+    {
+        link: "https://de.linkedin.com/company/ape-unit",
+        content: "Linkedin",
+    },
+]
+
+export default anchors;
```

Comment:
You can name this component as `socialMedias` as they are all social media links